docker/settings.dev.py

```diff
@@ -3,7 +3,7 @@
 """
 from typing import List
 
-GATEWAY_URL: str = 'http://unstable--dummy-api.libero.pub'
+GATEWAY_URL: str = 'http://unstable--api-gateway.libero.pub:8081'
```

Comment:
relies on actual data being there, whereas the `dummy-api` should always have some static data available; no particular preference as it's just for `dev`

Reply:
didn't realise that the dummy api is still up. I'll switch it back. It would be better to have a local content-store with static data.